unetsocket/c/test/test_unet.c

```diff
@@ -65,6 +65,9 @@ int main(int argc, char* argv[]) {
   int port_rx = 1100;
   uint8_t data[7] = {1,2,3,4,5,6,7};
   float* range = calloc(1,sizeof(float));
```

Comment:
Whoops.. Just saw these.. 

Instead of `calloc`-ing statically sized arrays, can't we just have an standard C types here?

```
float range = 0;
...
 rv = unetsocket_get_range(sock_tx, 31, &zrange);
 if (rv == 0) printf("Range measured is : %f \n", range);
```

Reply:
Thanks for catching this @notthetup. Fixed in 8c5019a